content/how-to/install-webpack.md

```diff
@@ -0,0 +1,41 @@
+---
+title: How to Install webpack?
+---
+
+### Pre-requistes
+
+We assume you have `node` and `npm` already installed.
+
+### Global Installation
+
+``` sh
+npm install webpack -g
+```
+
+The `webpack` command is now available globally.
+
+However, this is not a recommended practice. This locks you down to a specific version of webpack and might fail in projects that use a different version. The next section tells you how to install webpack locally in a project.
+
+### Local Installation
+
+``` sh
+npm install webpack --save-dev
+
+npm install webpack@<version> --save-dev
+
+```
+
+If you are using npm scripts in your project, npm will try to look for webpack installation in your local modules for which this installation technique is useful
+```
+package.json
+
+scripts: {
+	start: webpack --config mywebpack.config.js
```

Comment:
I mean, it should look like

```json
"scripts": {
  "start": "webpack"
}
```

That's valid json. I also added `json` to make highlighting work.